integration-cli/docker_api_inspect_test.go

```diff
@@ -42,3 +43,33 @@ func (s *DockerSuite) TestInspectApiContainerResponse(c *check.C) {
 		c.Fatalf("Path of `true` should not be converted to boolean `true` via JSON marshalling")
 	}
 }
+
+func (s *DockerSuite) TestInspectApiImageResponse(c *check.C) {
+	runCmd := exec.Command(dockerBinary, "pull", "busybox")
+	out, _, err := runCommandWithOutput(runCmd)
+
+	if err != nil {
+		c.Fatalf("failed to pull image: %s, %v", out, err)
+	}
+
+	runCmd = exec.Command(dockerBinary, "images", "-q", "busybox")
+	out, _, err = runCommandWithOutput(runCmd)
+
+	if err != nil {
+		c.Fatalf("failed to get image: %s, %v", out, err)
+	}
+
+	imageID := strings.TrimSpace(out)
+
+	endpoint := "/images/" + imageID + "/json"
+	status, body, err := sockRequest("GET", endpoint, nil)
+
+	c.Assert(status, check.Equals, http.StatusOK)
+	c.Assert(err, check.IsNil)
+
+	var inspectJSON types.ImageInspect
+	if err = json.Unmarshal(body, &inspectJSON); err != nil {
```

Comment:
for extra bonus points... do a build, check tags on image, should be empty, tag it, check again, should have one tag, tag again,should have two.  :-)   Verifies that you actually get all tags for an image instead of just the first one.